src/pages/components/tag/accessibility.mdx

```diff
@@ -19,205 +19,70 @@ import {
+![An operational tag in the tab order, disclosing related tags.](images/tag-accessibility-operational.png)
 
-<Row>
-  <Column noGutterSm>
-    <StructuredListWrapper>
-      <StructuredListHead>
-        <StructuredListRow head>
-          <StructuredListCell head>Environment</StructuredListCell>
-          <StructuredListCell head>Results</StructuredListCell>
-        </StructuredListRow>
-      </StructuredListHead>
-      <StructuredListBody>
-        <StructuredListRow>
-          <StructuredListCell>
-            - Microsoft Windows 10
-            <br />
-            - Firefox version 68
-            <br />
-            - JAWS 18
-            <br />- Carbon React version 7.7.1
-          </StructuredListCell>
-          <StructuredListCell>
-            <strong>JAWS test:</strong>
-            <OrderedList>
-              <ListItem>
-                Navigate to the Tag. JAWS announces the tag text.
-              </ListItem>
-              <ListItem>
-                Navigate to the Tag with Filter. JAWS announces the tag text and
-                clear filter.
-              </ListItem>
-            </OrderedList>
-          </StructuredListCell>
-        </StructuredListRow>
-      </StructuredListBody>
-    </StructuredListWrapper>
-  </Column>
+</Column>
 </Row>
 
-### iOS screen reader tests
+## Developer considerations
 
-<Row>
-  <Column noGutterSm>
-    <StructuredListWrapper>
-      <StructuredListRow head>
-        <StructuredListCell head>Environment </StructuredListCell>
-        <StructuredListCell head>Results</StructuredListCell>
-      </StructuredListRow>
-      <StructuredListBody>
-        <StructuredListRow>
-          <StructuredListCell>
-            - iOS version 12.2 with VoiceOver
-            <br />
-            - Safari version 12.2
-            <br />- Carbon React version 7.7.1
-          </StructuredListCell>
-          <StructuredListCell>
-            <strong>VoiceOver test:</strong>
-            <OrderedList>
-              <ListItem>
-                Swipe Right to the tag. VO announces the tag text and main
-                landmark.
-              </ListItem>
-              <ListItem>
-                Navigate to the tag with filter. Swipe Right to the "X". VO
-                announces "Clear Filter Image. Possibly very sharp close."
-                (Note: There is an open issue. VO should announce this as a
-                clear filter button.)
-              </ListItem>
-            </OrderedList>
-          </StructuredListCell>
-        </StructuredListRow>
-      </StructuredListBody>
-    </StructuredListWrapper>
-  </Column>
-</Row>
+Keep this in mind if you’re modifying Carbon or creating a custom component.
+
+- Do not add an `onClick` functionality to the dismissible tag, and only reserve
+  interactions for the close icon in the tag.
+- Do not nest buttons within the operational tag. Consider using
+  `PolymorphicProps` to change an element tag to avoid nesting buttons.
```

Comment:
I would write like that: 
```suggestion
- Do not nest buttons within the operational tag. Consider using
  `PolymorphicProps` to change an element tag to avoid nesting buttons.
```

Reply:
Thank you!